src/main/java/frc/robot/Robot.java

```diff
@@ -25,6 +27,13 @@
 
   private RobotContainer m_robotContainer;
 
+  //network table for vision control
+  private VisionCtrlNetTable VisionControlNetTable;
```

Comment:
Sorry to have lead you down the wrong path, but for the variable name by convention we use a lower case letter.